packages/relayer/src/logger.rs

```diff
@@ -0,0 +1,43 @@
+use file_rotate::{
+    compression::Compression,
+    suffix::{AppendTimestamp, FileLimit},
+    ContentLimit, FileRotate,
+};
+use lazy_static::lazy_static;
+use slog::{o, Drain, Level};
+use slog_async;
+use slog_json;
+use slog_term;
+
+lazy_static! {
+    pub static ref LOG: slog::Logger = init_logger();
+}
+
+fn init_logger() -> slog::Logger {
+    let log_terminal_decorator = slog_term::TermDecorator::new().build();
```

Comment:
Is it possible to use json format for log output to terminal when this relayer running on GCP instance?
Currently, GCP cloud logging saves docker's standard output as it is, so the logs for GCP cloud logging should be in json format.

For example, if `logger=json` is passed in the docker startup option, the desired behavior is for standard output to be json-style.
Or we can set logger style env in .env file.

Reply:
Implemented, there is a env variable called `JSON_LOGGER` which you can set to true (case insensitive) for logging in json format.

Reply:
@Bisht13 
Thanks for your quick work!